build.sbt

```diff
@@ -2226,7 +2226,6 @@ lazy val `runtime-compiler` =
       annotationProcSetting,
       (Test / fork) := true,
       libraryDependencies ++= Seq(
-        "com.chuusai"     %% "shapeless"               % shapelessVersion   % "provided",
```

Comment:
I don't see any other usages of `shapelessVersion`. I guess:
```
val shapelessVersion        = "2.3.10"
```
should also be removed then.

Reply:
Yes, thanks